build.sbt

```diff
@@ -0,0 +1,40 @@
+import sbt._
+
+name := "sauna"
+
+version := "0.0.2"
+
+scalaVersion := "2.11.7"
+
+lazy val `sauna-server` = project in file(".")
+
+val playVersion = "2.4.4"
+
+libraryDependencies ++= Seq(
+  "com.github.seratch" %% "awscala" % "0.5.+",
+  "com.typesafe" % "config" % "1.3.0",
+  "com.typesafe.play" %% "play-json" % playVersion,
+  "com.typesafe.play" %% "play-ws" % playVersion,
+
+  "org.scalatest" %% "scalatest" % "2.2.4" % "test"
+)
+
+javacOptions ++= Seq(
+  "-source", "1.8",
+  "-target", "1.8",
+  "-Xlint"
+)
+
+scalacOptions ++= Seq(
+  "-deprecation",
+  "-encoding", "UTF-8",
+  "-feature",
+  "-unchecked",
+  "-Xfatal-warnings",
+  "-Ywarn-dead-code",
+  "-Ywarn-numeric-widen",
+  "-Ywarn-value-discard",
+  "-Ywarn-unused-import"
+)
+
+ivyScala := ivyScala.value map { _.copy(overrideScalaVersion = true) }
```

Comment:
Can we switch from a build.sbt to .scala files (see e.g. https://github.com/snowplow/snowplow/tree/master/3-enrich/scala-hadoop-enrich/project)